### ISSUE TITLE
can't resolve ./config in node-mailjet when using docker

### ISSUE THREAD

**Issue by javierguzman** [mailjet/mailjet-apiv3-nodejs#121]
Hello all,

I have been running my development server locally without any problem for a while and now I am trying to make a docker compose version of it. However, I get the following error:

```
ERROR in /opt/app/node_modules/node-mailjet/mailjet-client.js
Module not found: Error: Can't resolve './config' in '/opt/app/node_modules/node-mailjet'
 @ /opt/app/node_modules/node-mailjet/mailjet-client.js 211:17-36
 @ /opt/app/node_modules/node-mailjet/index.js
```
This error does not happen locally. The only difference I can think of is that when I run locally my node_modules are in the same level as package.json and src folder, etc. However, when I run docker compose I have node_modules in the parent folder and in a folder underneath I have the src folder. This is a well known solution for being able to run locally as well as in docker without mixing node_modules from host and from docker.

Any idea of what the problem could be?

Thank you in advance and regards.

**Comment by tonchis** [comment]
@javierguzman 👋 can you please share your `Dockerfile` and `docker-compose.yml`, please?

**Comment by javierguzman** [comment]
Hello @tonchis! Thanks for the response. I have placed them in the following link, please, let me know if you need something else.
https://codesandbox.io/s/node-mailer-error-8jp2i

Dockerfile is under server/config and docker-compose.yml is under common.

One important thing I have noticed is that if I remove the code that uses note-mailjet, then all works ok, so that makes me even more suspicious about the node-mailjet code.

**Comment by javierguzman** [comment]
On the following stackoverflow question some people mention that the problem might be caused due to the lack of the semicolon at the end of the require.

https://stackoverflow.com/questions/33007878/nodejs-typeerror-require-is-not-a-function

Down the error I placed above, I get also this:
TypeError: require is not a function

So maybe that is the problem??

**Comment by javierguzman** [comment]
I resolved the json stuff by adding json to the webpack extension fields:
extensions: ['.ts', '.js', '.json']

However, I still have the require is not a function issue which seems to be related to #92

**Comment by javierguzman** [comment]
""Resolved"" by just removing this module and making directly API queries with axios. A pity but I close this.

**Comment by scroll17** [comment]
Hey @javierguzman @HarmNullix 
Published release `3.3.8` with fixed this problem
